src/logutil/logger.h

```diff
@@ -55,6 +55,8 @@ class Logger
   static void LogInfo(const string& message);
   static void LogWarn(const string& message);
   static void LogError(const string& message);
+  static string s;
+  friend class LoggerImpl;
```

Comment:
Again, the approach is not correct. Let's revisit the requirement again. I've wrote it in the PR.